Data/Generated.lua

```diff
@@ -103,3 +103,120 @@ for index, name in ipairs(keystones) do
 end
 table.insert(skinOfTheLords, "Corrupted")
 table.insert(data.uniques.generated, table.concat(skinOfTheLords, "\n"))
+
+--[[ 3 scenarios exist for legacy mods
+	- Mod changed, but kept the same mod Id
+		-- Has legacyMod
+	- Mod removed, or changed with a new mod Id
+		-- Has only a version when it changed
+	- Mod changed/removed, but isn't legacy
+		-- Has empty table to exclude it from the list
+]]
+local watchersEyeLegacyMods = {
```

Comment:
Consider just providing the legacy values and handling the substitution in the caller instead.

Reply:
The reason I did it this way was to remain flexible in case other substitutions are needed when mods change (e.g. if a mod is added that gets two separate rolls).  It so happens that these all use the same type of substitution.